app/components/ButtonsGroup/ButtonsGroup.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function ButtonsGroup({ buttons, }) {
+  const style = {
+    height: `${100 / buttons.length}%`,
+  };
+  function createButton(buttonObject, index) {
```

Comment:
Function returns a react component, so it should adhere to the naming convention: start with a Capital and not a verb. I know it's confusing, since it is also a function. But in JS most things are functions, and the function naming convention is a good baseline, which has exceptions for some cases. Some of these exceptions are in plain JS, others are for frameworks.
Examples:
 - Before ES2015, classes were functions, so `function Animal(){}` is a thing, as long as that's a class.
 - Factories (which return instances of different classes), are also functions and start with a Capital: `function Vehicle()`, which can return a Car or Boat or Spaceship.
 - React stateless components :)

Reply:
so in this specific case `Button()` would make sense?

would it also require moving it to a separate file..? (cause then I suspect that `Button()` would be a bit too general)

Reply:
Yes, if inside the file, you can use `Button`. I'd call it _Some_ Button and extract to a new file. GroupButton? AppButton?
If generic, Button is good. If specific, should be "SomeSpecificButton".

Reply:
done